### ISSUE TITLE
Upgrade to 2.5.5 and removed header shadow

### ISSUE THREAD

**Issue by AlirezaAkbarix** [react-navigation/react-navigation#4607]
Just upgrade package from `2.3.1` to the latest version (`2.5.5`) and the shadow below header is gone!

`2.3.1`:
![image](https://user-images.githubusercontent.com/15062651/42121364-e18845de-7c42-11e8-8ace-e1c09b7e859f.png)

`2.5.5`:
![image](https://user-images.githubusercontent.com/15062651/42121175-50158c90-7c3f-11e8-995e-15be570250a7.png)

Please don't notice issue is not respecting the issue template as it's a clear issue.

Environment:
  OS: macOS High Sierra 10.13
  Node: 8.9.4
  Yarn: 1.6.0
  npm: 6.0.0
  Watchman: 4.9.0
  Xcode: Xcode 9.2 Build version 9C40b
  Android Studio: 3.1 AI-173.4720617

Packages: (wanted => installed)
  react: ^16.4.1 => 16.4.1
  react-native: 0.55.4 => 0.55.4

**Comment by brentvatne** [comment]
can you please post a minimal example that reproduces this?

**Comment by ramsanath** [comment]
I am facing the same issue 

Steps to reproduce

Create a StackNavigator with a basic screen

>  
    import {createStackNavigator} from 'react-navigation';
    import React, {Component} from 'react'
    import {View} from 'react-native'
    
    class Demo extends Component {
        render() {
            return (
                <View
                    style={{
                        flex: 1,
                        backgroundColor: 'powderblue'
                    }}
                />
            )
        }
    }
    
    const RootNavigator = createStackNavigator({
        Home: Demo,
    });
    
    export default RootNavigator;

What is expected:

![expected](https://user-images.githubusercontent.com/25882182/42167993-63be3432-7e2d-11e8-811e-379990c6d359.png)

Actual output:

![screenshot_1530539481](https://user-images.githubusercontent.com/25882182/42168021-78c38bfc-7e2d-11e8-933b-e4a139c908ee.png)

**Info**
platform: OSX
react-navigation: ^2.5.5
react-native: 0.55.4

**Comment by kolyneh** [comment]
Have the same issue on 2.5.5.

**Comment by kidmysoul** [comment]
Have the same issue on 2.5.5.

**Comment by xanderberkein** [comment]
Snack: https://snack.expo.io/rkGrH3dMX

The shadow disappeared in 2.5.4, might be because of this commit https://github.com/react-navigation/react-navigation/commit/54448ed070d87b590cb5f7c9bc2c7bd93a126714, which touched the header?

**Comment by ramsanath** [comment]
Thanks @xanderberkein 

Your suspicion appears to be true.

I experimented with the default header colour and  found that the default colour value for android '#FFF' was causing the problem.
I replaced it to '#FFFFFF' and I got the shadows back.

Specifying it it headerOptions like this is not helping.

>
	{
	    navigationOptions: {
	      headerStyle: {
	        backgroundColor: '#FFFFFF',
	      },
	      headerTintColor: '#FFFFFF',
	    },
	}

I doubted short hand hex colours but couldn't find a valid reason.

So the workaround is to edit the lib file at 
`node_modules/react-navigation/src/views/Header/Header.js`
at line number 546 
from `... : '#FFF';` to `... : '#FFFFFF';`

Example: 

![screen shot 2018-07-03 at 20 00 35](https://user-images.githubusercontent.com/25882182/42226227-0a9aeb82-7efc-11e8-936b-e6dc248b80ae.png)
